<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>481</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>You will be using a voice password, click start recording and say the word you will like to use as a password. Click stop recording to stop the recording. (Please note that your password should be at least 8 letters long)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="password">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>401</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(64, 64, 64, 19);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="startRecordingButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border-radius: 14px;
color: white;
background-color: rgb(0, 0, 13);
font-weight: bold;
font-size: 15px;
font: 10pt &quot;Arial&quot;;
}
QPushButton:hover{
background-color: rgb(0, 0, 25); 
color: rgb(0, 153, 255);
font-weight: bold;
}
</string>
   </property>
   <property name="text">
    <string>Start Recording </string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopRecordingButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border-radius: 14px;
color: white;
background-color: rgb(51, 51, 51);
font-weight: bold;
font-size: 15px;
font: 10pt &quot;Arial&quot;;
}
QPushButton:hover{
background-color: rgb(64, 64, 64); 
color: rgb(0, 153, 255);
font-weight: bold;
}
</string>
   </property>
   <property name="text">
    <string>Stop Recording</string>
   </property>
  </widget>
  <widget class="QLabel" name="listening_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
